src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -90,4 +99,37 @@ private String setDisplayCaseReferenceFlag(String caseRef, Classification classi
             .map(CcdCollectionElement::new)
             .collect(toList());
     }
+
+    private String extractSurnameFromOcrData(Envelope envelope) {
+        if (CollectionUtils.isEmpty(envelope.ocrData)) {
+            return null;
+        }
+
+        String surnameOcrFieldName = serviceConfigProvider.getConfig(envelope.container)
+            .getSurnameOcrFieldName(envelope.formType);
+
+        List<String> surnameList = envelope.ocrData.stream().filter(ocrData -> ocrData.name.equals(surnameOcrFieldName))
+            .map(ocrData -> ocrData.value).collect(Collectors.toList());
+        if (surnameList.size() == 0) {
+            LOGGER.info(
+                "Surname not found in OCR data. Surname Ocr Field Name:{}. Envelope id:{},Case Ref:{},Jurisdiction:{}",
```

Comment:
personally i would consider this as a debug, but don't take my word for it. see what team thinks of it.

p.s. please put single space after `:` and `,`

Reply:
agree but for now we can keep is info as we will need this log while testing definitions